sklearn/cluster/_affinity_propagation.py

```diff
@@ -469,7 +469,7 @@ def fit(self, X, y=None):
             target_type=numbers.Real,
             min_val=0.5,
             max_val=1,
-            closed="right",
```

Comment:
the description of the param says "between 0 and 1". If 1 is excluded, maybe we should mention it there

Reply:
let me second check the previous code to be sure but I think it was excluded.

Reply:
my bad the interval is 0.5 < dumping <= 1.0

Reply:
hum, but 0.5 is the default so it should be included :)

Reply:
FYI: https://github.com/scikit-learn/scikit-learn/pull/20723/files#diff-62083de22888eadb572404f8f7255a19a74370eeaf2a893858b066d90ada979eL135-L136

Reply:
> hum, but 0.5 is the default so it should be included :)

Yes, I edited my answer. I confused the valid interval and the condition used to raise the error :)